packages/shi/src/cli.ts

```diff
@@ -0,0 +1,146 @@
+import { parseArgs } from "util";
```

Comment:
`util` is built into NodeJS so it doesn't generally require an entry in `package.json` dependencies, so I don't think there's really anything to change. But it makes me think about our approach to documenting these sorts of dependencies, as it could effect which versions of Node our project is compatible with. 

I imagine this could at some point become a table of dependencies and node versions, mapping out which features are supported by which node versions, to give more visibility/clarity to our Node. version support.